articles/howto-build-astatic-site-with-gatsbyjs/index.md

```diff
@@ -0,0 +1,281 @@
+
+
+### What is GatsbyJS?
+
+>GatsbyJS can be defined as a static site generator that uses [React.js](https://reactjs.org/) (for the client-side) and [GraphQL](https://graphql.org/) (to access data) to build a reliable and faster website.
+
+A software application that generates HTML pages from templates and a given content source is called a “static site generation”. Markdown-formatted text files are used by most static site generators.
+Static site generators are different from content management systems such as WordPress and Drupal that are powered by databases. 
+List of other static site generators for JAMStack Sites includes [Next.Js](https://nextjs.org/), [Hugo](https://gohugo.io/), [Jekyll](http://jekyllrb.com/), [Hexo](https://hexo.io/), [Slate](https://slatedocs.github.io/slate/), [NuxtJS](https://nuxtjs.org), [GitBook](https://www.gitbook.com/), [Docusaurus](https://v2.docusaurus.io/) and [more](https://jamstack.org/generators/) 
+
+**Advantages of GatsbyJS**
+- It's [faster](https://www.gatsbyjs.com/contributing/how-to-pitch-gatsby/) to create a static site with GatsbyJs.
+- It's easy to set up.
+- It's scalable.
+- It has a great [community of developers](https://www.gatsbyjs.com/contributing/community/).
+
+### Prerequisites
+
+To get started with GatsbyJS, you will need some background knowledge on the following:
+- [React.js](https://reactjs.org/) (the basics, components, etc.)
+- [Node.js](https://nodejs.org/en/) (the basics, [npm](https://www.npmjs.com/), etc)
+- [GraphQL](https://graphql.org/) (the basics and advanced)
+
+However there’s no problem if you don’t have any knowledge of these technologies, this article will help you get started with GatsbyJS, so stick around and check the links out later if you are interested in advancing your knowledge.
+
+### Installations
+
+First of all, you'll need to install [Node.js](https://nodejs.org/en/) on your computer.
+Go to the [official Node.js website](https://nodejs.org/en/) to download the Node.js version for your operating system.
+
+After installing it, open up your terminal and type in:
+
+```bash
+node –version
+```
+
+and
+
+```bash
+npm –version
+```
+
+These commands will give you the version of Node.js and node package manager respectively.
+
+Next, in your terminal type in:
+
+```bash
+npm install –global gatsby-cli
+```
+
+You just installed Gatsby and its command-line interface globally on your computer, this is the first step to interact with Gatsby using the command line interface.
+
+In order to confirm you have Gatsby installed, type in:
+
+```bash
+gatsby –version
+```
+
+### Creating a new GatsbyJS site
+
+Now let's create a static site using GatsbyJS.
+
+Open up the terminal, in your code editor if you are using [VS Code](https://code.visualstudio.com/), It comes built-in with a terminal. You will be using the GatsbyJS site template from GitHub to create your site from scratch.
+
+Create a **my-first-gatsby-site** folder, and then in your terminal, type in
```

Comment:
How to create the folder? `mkdir` command missing or clone from Github command since Github is mentioned above?

Reply:
@Jethro-magaji 
Just wanted to ping you on this comment - not sure if I see that this step was added?

Reply:
Done